app/views/attendances/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(@attendance) do |f| %>
+  <% if @attendance.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@attendance.errors.count, "error") %> prohibited this attendance from being saved:</h2>
+
+      <ul>
+      <% @attendance.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :date %><br>
+    <%= f.text_field :date %>
+    <%= f.label :present %><br>
+    <!-- #Don't know what to put here -->
```

Comment:
What you have here is fine for present. The thing I think you're missing is some way to connect this to a student and a course. I would say use the same technique for dropdowns that you used in CourseLoads